style.css

```diff
@@ -435,11 +435,10 @@ body {
   margin-top: 10px;
 }
 
-
 .button1:hover {
   background-color: rgb(127 140 255 / 100%);
   color: rgb(255 255 255 / 100%);
```

Comment:
- Hey, since this is your last review, I'd approve your PR but please, update this color to `#4053fc` which is the correct color for the hover. kindly fix it before you merge. :pray:

Reply:
ok thanks